src/utils.ts

```diff
@@ -400,11 +400,16 @@ export async function launchBrowser(browserPath: string, port: number, targetUrl
     ];
 
     const headless: boolean = isHeadlessEnabled();
+    const corsEnabled: boolean = isCorsEnabled();
 
     if (userDataDir) {
         args.unshift(`--user-data-dir=${userDataDir}`);
     }
 
+    if (corsEnabled) {
+        args.unshift('--disable-web-security');
```

Comment:
`--disable-web-security` disables same-origin policy, which isn't the same as CORS iirc. Does launching the browser with this switch address your usecase? Additionally, if CORS is enabled we don't want to pass this switch, so the check should be inversed.

Reply:
yes, `--disable-web-security` works for me.
ok, i changed a bit, using an array as browserArgs now.  should meet the feature.

Reply:
![image](https://user-images.githubusercontent.com/995572/141453411-b09c76fe-574c-4d69-9be9-6b3e6ca756c6.png)

Reply:
Great. Update the title of the PR to reflect the new work and I'll approve :)

Reply:
updated